i18n/en.json

```diff
@@ -3914,6 +3914,7 @@
   "signup.office365": "Office 365",
   "signup.saml.icon": "SAML Icon",
   "signup.title": "Create an account with:",
+  "someting.string": "defaultString",
```

Comment:
Looks like a historical typo? "something"?

Reply:
Yeah, it's a historical typo. This string is actually just in a mattermost-redux unit test, not in the actual code, so I'm not too concerned about fixing it right now. We should probably update `i18n-extract` at some point to not look at test files.